runtime/devnet/src/config/api.rs

```diff
@@ -0,0 +1,32 @@
+use crate::{config::assets::TrustBackedAssetsInstance, fungibles, Runtime, RuntimeCall};
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::traits::Contains;
+
+#[derive(Encode, Decode, Debug, MaxEncodedLen)]
+#[repr(u8)]
+pub enum RuntimeStateKeys<T: fungibles::Config> {
```

Comment:
```suggestion
/// A query of runtime state.
pub enum RuntimeRead<T: fungibles::Config> {
```

Align with terminology.